### ISSUE TITLE
Change language removes NPPTextFX2 menu

### ISSUE THREAD

**Issue by schnaggi** [rainman74/NPPTextFX2#20]
Hi,

this is my debug view
Notepad++ v8.5.8   (64-bit)
Build time : Oct 15 2023 - 21:43:56
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Enterprise (64-bit)
OS Version : 21H2
OS Build : 19044.3570
Current ANSI codepage : 1252
Plugins : 
    ComparePlus (1.1)
    CsvQuery (1.2.9)
    DSpellCheck (1.5)
    Hexeditor (0.9.12)
    JSMinNPP (1.2308)
    LuaScript (0.12)
    mimeTools (2.9)
    NppConverter (4.5)
    NppExport (0.4)
    NppTextFX (1.4.1)
    XMLTools (3.1.1.13)

I used the 64bit version
https://github.com/rainman74/NPPTextFX2/releases/download/1.4.1/NppTextFX2.1.4.1.x64.zip

I have the latest language versions of german and english language pack installed.

Issue:
My default language is German,  If I open Preferences and change language to English, most menu entries are translated and TextFX menu disappear . If I change back the language the menu is still gone and only recover if I restart NPP.
I also tested English as default and German as alternative language but there is no difference.

No I don't know if this is NPP or TextFX issue. The issue also occur if I disable all additional plugin.

I will add three snapshots later to show the issue.

**Comment by rainman74** [comment]
Yes, I can confirm this behavior, but it's more likely due to Notepad++ itself, as it no longer reinitializes the plugin. However, since the menu reappears permanently after a brief app restart, that's sufficient for me.
